include/aws/event-stream/event_stream.h

```diff
@@ -81,7 +83,6 @@ enum aws_event_stream_header_value_type {
     AWS_EVENT_STREAM_HEADER_UUID
 };
 
-static const uint16_t UUID_LEN = 16U;
```

Comment:
would defining it as extern mitigate the warning?

Reply:
actually nevermind. we are probably better off hiding this constant. name is to generic and can break someone